sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -48,6 +49,8 @@
     <Compile Include="Shared\AzureKeyCredentialPolicy.cs" />
     <Compile Include="Shared\AzureSasCredentialSynchronousPolicy.cs" />
     <Compile Include="Shared\Base64Url.cs" />
+    <Compile Include="Shared\ChangeTrackingList.cs" />
```

Comment:
I don't see these classes used anywhere can we remove the two change tracking files here?

Reply:
They were added due to `Optional` depends on them, removed now.